docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  postgres:
```

Comment:
we should probably mount these services to a directory so that the data is retained.

Reply:
I wouldn't use docker-composer for production

Reply:
@ptman even while working in local. one might need the test data retained right?

Reply:
It's not like anyone is forbidden from customizing docker-compose.yml to their needs. What makes sense as a default for trying out chatwoot in a test environment?